Dockerfile

```diff
@@ -1,5 +1,5 @@
 ARG APP_INSIGHTS_AGENT_VERSION=3.2.8
```

Comment:
update this too please https://github.com/microsoft/ApplicationInsights-Java/releases